test/lib/creation.go

```diff
@@ -114,6 +114,16 @@ func (client *Client) CreateConfigMapPropagationOrFail(name string) *configsv1al
 	return configMapPropagation
 }
 
+// CreateConfigMapOrFail will create a configmap or fail the test if there is an error.
+func (client *Client) CreateConfigMapOrFail(name, namespace string, data map[string]string) *corev1.ConfigMap {
+	configMap, err := client.Kube.Kube.CoreV1().ConfigMaps(namespace).Create(resources.ConfigMap(name, data))
```

Comment:
Please add `client.T.Logf("Creating configmap %s", name)` before this line. Not important, but would be helpful for debugging if there is an error.